step.yml

```diff
@@ -65,6 +65,15 @@ inputs:
 
         Call `xcodebuild -help` for available export options.
       category: "app/pkg export configs"
+  - xcodebuild_options:
+    opts:
+      category: Debug
```

Comment:
Adding at the end of the ""xcodebuild configs" category would be more useful than a single element "Debug" category.

Reply:
should we also change this for the xcode-archive step?

Reply:
No as the ios and mac export step inputs have different structure.